### PR TITLE
:meat_on_bone: added columns in css for filters

### PR DESCRIPTION
**Pull request by ricardoom**
very basic columns set up for the filters

### CODE REVIEW

File: docs/discovery-search.html [hunk 1]
```diff
@@ -572,8 +572,23 @@ <h3 id="filter-title">Filter Results</h3>
                   <label for="language-filter-4">Language</label>
               </li>
               <li>
-                <input id="language-filter-5" type="checkbox" name="language" value="Language" />
-                <label for="language-filter-5">Language</label>
+                <input id="language-filter-6" type="checkbox" name="language" value="Language" />
```
Comment (EdwinGuzman) [inline docs/discovery-search.html]: Did you intentionally skip `language-filter-5`?

Comment (EdwinGuzman) [reply docs/discovery-search.html]: Okay, doesn't look like it matters though.

File: sass/_general-mixins.scss [hunk 2]
```diff
@@ -101,6 +101,8 @@ $bullet-size: 0.5rem;
   border: 0;
 }
 
+
+//
```
Comment (EdwinGuzman) [inline sass/_general-mixins.scss]: Remove.

**Comment by ktp242** [comment]
Just need the minor changes as Edwin mentioned!

File: docs/discovery-search.html (hunk 1 above)
Comment (ricardoom) [reply docs/discovery-search.html]: added it back

File: sass/_general-mixins.scss (hunk 2 above)
Comment (ricardoom) [reply sass/_general-mixins.scss]: gone

**Comment by ricardoom** [comment]
mergetastic. @EdwinGuzman go to bed, you've got to be in on time tomorrow, or else!

**Comment by EdwinGuzman** [comment]
🐶

**Pull request merged**